dynamic_instrumentation/src/instrument.rs

```diff
@@ -428,7 +424,7 @@ fn instrument_body<'a, 'tcx>(
     body_did: DefId,
 ) {
     let hooks = Hooks::new(tcx);
-    let mut adder = InstrumentationAdder::new(tcx, hooks, body);
+    let mut adder = InstrumentationPointCollectorVisitor::new(tcx, hooks, body);
```

Comment:
This variable name should also be updated, perhaps to `visitor`, `collector`, or `collect_points`.